addons/test_website/models/model.py

```diff
@@ -20,10 +20,11 @@ class TestModel(models.Model):
     _description = 'Website Model Test'
 
     name = fields.Char(required=True)
+    sub_ids = fields.One2many('test.submodel', 'parent_id', "Submodels")
```

Comment:
`sub_ids`? We would not named a field like that in regular business code, let's consider test code as regular code please :pray: